### PR TITLE
feat: Add CNV deployment script

### LINKED ISSUE #8: Deploy CNV
The README file shows how to deploy kubevirt instead CNV

### PR DESCRIPTION
**Pull request by tripledes**
* The script deploys CNV through OLM
* The only requirement should be a working OCP4 with OLM and a working pull secret
* Fixes #8

### CODE REVIEW

**Comment by tripledes** [comment]
I just noticed the main README has instructions for kubevirt, I'll amend that and remove the WIP from the title

File: CNV/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Container Native Virtualization Deployment
+
+## What does it do
+
+The script [deploy-cnv.sh](deploy-cnv.sh) deploys CNV through [OLM](https://github.com/operator-framework/operator-lifecycle-manager), meaning:
```
Comment (rthallisey) [inline CNV/README.md]: Keep in mind this is CNV 2.0.   Might be worth making a note about it.

Comment (tripledes) [reply CNV/README.md]: Added. Thanks!

**Comment by markmc** [comment]
I think this is a fine start, thanks! 

I don't love all this waiting on pod conditions - as a follow-up, I think we should wait on more specific resource status than pod conditions

Also, I think we're going to need to switch to pre-release CNV 2.1 builds soon - I've filed openshift-kni/install-scripts#12 for that

**Pull request merged**